target/linux/ramips/image/mt7620.mk

```diff
@@ -661,6 +667,17 @@ define Device/netgear_wn3000rp-v3
 endef
 TARGET_DEVICES += netgear_wn3000rp-v3
 
+define Device/netis_wf2770
+  MTK_SOC := mt7620a
```

Comment:
Rename to "SOC" due to 58e71e322bc0e0edd08157075e9022c15ae33ec5.

Reply:
Done.